tools/autograd/derivatives.yaml

```diff
@@ -2702,7 +2702,7 @@
   cpu_nested_shape_example: non_differentiable
 
 - name: to_padded_tensor(Tensor self, float padding, int[]? output_size=None) -> Tensor
-  self: at::_nested_from_padded(grad, self._nested_tensor_size())
+  self: at::_nested_from_padded(grad.contiguous(), self._nested_tensor_size())
```

Comment:
This can be removed now?

Reply:
Yes this can be removed now. Do you think having it is over cautious? Or do we have to trust the contract that nested_from_padded provides hence we don't need it, and adding it is confusing.

Reply:
I think it is confusing because it gives the false impression that this function requires contiguous inputs.

Reply:
Yeah that makes sense, just pushed the change